frontend/src/registration/components/InstanceSetupPage/InstanceSetupPage.tsx

```diff
@@ -30,18 +31,22 @@ interface Props extends StateProps, ActionProps {
         submitRegistration: submitRegistration,
     }
 )
+
 export class InstanceSetupPage extends React.PureComponent<Props> {
     render() {
         return (
             <RegistrationPage title="Create your Pro & Teacher Account"
                               currentStep={2}>
-                <Jumbotron>
-                    <p>
+                <Jumbotron className="domain-available">
```

Comment:
There are two types of this jumbotron:
1. domain available,
1. domain connected.

Do you think it would be reasonable to extract it to a separate component (and maybe pass `messages` there as a prop) to ensure that the consistency is maintained between them (in case of any changes)?

Reply:
Sure, I've created `DomainSuccessJumbotron` to handle this text change. 
I'm not sure how to add tests for this tough. :/

Reply:
Manage to add tests to the component! Yaay.
I've learned something today :grin: